apps/snmptrapd_kafka.c

```diff
@@ -195,23 +196,20 @@ static netsnmp_kafka_globals _kafka = {
 /* FW */
 int kafka_handler(netsnmp_pdu *pdu,netsnmp_transport *transport,netsnmp_trapd_handler *handler);
 
-static void
```

Comment:
Can't change function declaration style, other parts of net-snmp use the previous one.